app.go

```diff
@@ -0,0 +1,24 @@
+package versionbundle
+
+import "encoding/json"
+
+type App struct {
+	App              string `yaml:"app"`
+	ComponentVersion string `yaml:"componentVersion"`
+	Version          string `yaml:"version"`
+}
+
+func CopyApps(apps []App) []App {
```

Comment:
```suggestion
func CopyApps(apps []App) []App {
    appList := make([]App, len(apps))
    copy(appList, apps)
    return appList
}
```

I find the JSON mangling foor deep copy purpose a bit weird. When there are no internal pointers & such involved, I think the builtin `copy` is more expressive to make clear what the function does :slightly_smiling_face:

Reply:
You're right, it's weird to marshaling only for copying them. 
`CopyBundle` is doing the same thing but I think that's because it has a nested structure. 
https://github.com/giantswarm/versionbundle/blob/bb309403328a6b8ee85d88e2774a5cc902a4f36c/bundles.go#L63